src/components/FilmsList.vue

```diff
@@ -1,23 +1,35 @@
 <template>
   <div>
-    <div class="d-flex flex-column justify-content-center align-items-end container" v-if="!loading">
+    <div
+      class="d-flex flex-column justify-content-center align-items-end container mt-3"
+      v-if="!isLoading"
+    >
       <div class="d-flex flex-row">
         <b-list-group horizontal class="flex-sm-wrap">
           <b-list-group-item
-              class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
-              v-for="movie in uniqueMovies"
-              :key="movie.id"
+            class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
+            v-for="{
+              title,
+              backdrop_path,
+              overview,
+              original_language,
+              original_title,
+              release_date,
+              vote_average,
+              id,
+            } in uniqueMovies"
+            :key="id"
           >
             <film-card
-                :title="movie.title"
-                :backdropPath="movie.backdrop_path || ''"
-                :overview="movie.overview"
-                :originalLanguage="movie.original_language"
-                :originalTitle="movie.original_title"
-                :releaseDate="movie.release_date || ''"
-                :overallRating="movie.vote_average"
-                :cardId="movie.id"
-                @getCardId="getMovieDetails"
+              :title="title"
+              :backdropPath="backdrop_path || ''"
+              :overview="overview"
+              :originalLanguage="original_language"
+              :originalTitle="original_title"
+              :releaseDate="release_date || ''"
+              :overallRating="vote_average"
+              :cardId="id"
+              @getCardId="getMovieDetails"
```

Comment:
When naming events use `kebab-case` i.e. instead of naming the event "getCardId" call it "get-card-id".

All JS event names follow this convention.